proposals/0320-clock-instant-date-duration.md

```diff
@@ -131,98 +108,86 @@ public protocol Clock: Sendable {
   
   var now: Instant { get }
   
-  func sleep(until deadline: Instant) async throws
+  func sleep(until deadline: Instant, tolerance: Instant.Interval?) async throws 
   
-  var minimumResolution: Duration { get }
+  var minimumResolution: Instant.Interval { get }
   
   func measure(_ work: () async throws -> Void) reasync rethrows -> Duration
 }
 ```
 
-This means that given an instant, it is intrinsically linked to the clock; e.g., a monotonic instant is not meaningfully comparable to a wall clock instant. However, as an ease of use concession, the durations between two instants can be compared. However, doing this across clocks is considered a programmer error, unless handled very carefully. By making the protocol hierarchy just clocks and instants, it means that we can easily express a compact form of a duration that is usable in all cases; particularly for APIs that might adopt Duration as a replacement to an existing type.
+This means that given an instant, it is intrinsically linked to the clock; e.g., a specific clock's instant is not meaningfully comparable to all other clock instants. However, as an ease of use concession, the durations between two instants can be compared. However, doing this across clocks is potentially considered a programmer error, unless handled very carefully. By making the protocol hierarchy just clocks and instants, it means that we can easily express a compact form of a duration that is usable in all cases; particularly for APIs that might adopt Duration as a replacement to an existing type.
 
-The clock minimum resolution will have a default implementation that returns `.nanosecond(1)`. 
+The clock minimum resolution will have a default implementation that returns `.nanosecond(1)`. This property serves to inform users of a clock the potential minimum granularity of what to invocations to now may return but also indicate the minimum variance between two instants that are significant. Practically speaking, this becomes relevant when measuring work - execution of a small work load may be executed in under the minimum resolution and not provide accurate information. 
 
-Clocks can then be used to measure a given amount of work. This means that clock should have the extensions to allow for the affordance of measuring workloads for metrics, but also measure them for performance benchmarks. In addition to the per clock definitions of measuring a base measurement function using the monotonic clock will also be added.
-
-```swift
-public func measure(_ work: () async throws -> Void) reasync rethrows -> Duration
-```
-
-This means that making benchmarks is quite easy to do:
+Clocks can then be used to measure a given amount of work. This means that clock should have the extensions to allow for the affordance of measuring workloads for metrics, but also measure them for performance benchmarks. This means that making benchmarks is quite easy to do:
 
 ```swift
 let elapsed = measure {
   someWorkToBenchmark()
 }
```

Comment:
This should be `someClock.measure`, since we don't have the `measure()` free function anymore?

Reply:
correct, that is a mistake - will include in the next batch of corrections